recipe/meta.yaml

```diff
@@ -42,11 +41,14 @@ test:
         - pystache-test
 
 about:
-    home: http://github.com/defunkt/pystache
+    home: http://github.com/sarnold/pystache
```

Comment:
@sarnold building from forks is usually frowned upon. What is the story here? Is the project dead? Could your changes be a patch instead? (Patches are part of the recipe and easier to inspect.)

Reply:
Sadly, no.

* upstream no activity on github at all
* old maintainer no pystache activity in 7 yrs (but still some on github)
* way too big for a patch

Reply:
This is very clean now, with almost complete test coverage:  https://github.com/sarnold/pystache

Reply:
@sarnold the core members are not very keen on "keep-alive" forks that are not the canonical repo or "blessed" by the original maintainer. Ideally, forks would change the name or request upstream to became the canonical source.

Reply:
I'm not sure what " request upstream " means exactly. I can make a PR and see what happens.